www-servers/sniproxy/sniproxy-0.6.0.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools user
+
+if [[ ${PV} == 9999* ]]; then
+	EGIT_REPO_URI="https://github.com/dlundquist/sniproxy.git"
+	EGIT_BRANCH="master"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/dlundquist/sniproxy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Proxies incoming HTTP and TLS connections based on the hostname"
+HOMEPAGE="https://github.com/dlundquist/sniproxy"
+
+LICENSE="BSD-2"
+SLOT="0"
+IUSE="+dns +largefile rfc3339"
```

Comment:
seems I missed that on the first review. sorry about that.

what about those useflags? 
I see no `src_configure` that uses them. this does not happen automatically, so you need to explicitly define configure function and pass appropriate flags.

please add something like
```
src_configure() {
        local myconf=(
           #use conditional code here
        )
        econf "${myconf[@]}"
}
```

or drop useflags completely if installing with default options is ok and those options do not change at configure time depending on system.

Reply:
Oh, I don't know what happen ... I was sure to have already do that for this ebuild...

```diff
--- a/www-servers/sniproxy/sniproxy-0.6.0.ebuild
+++ b/www-servers/sniproxy/sniproxy-0.6.0.ebuild
@@ -41,6 +41,16 @@ src_prepare() {
        sed -i "/create/s/daemon/sniproxy/" debian/logrotate.conf || die "Unable to replace logrotate configuration"
 }

+src_configure() {
+       local my_conf=(
+               $(use_enable dns)
+               $(use_enable largefile)
+               $(use_enable rfc3339 rfc3339-timestamps)
+       )
+
+       econf "${my_conf[@]}"
+}
+
 src_install() {
        default
```